lib/controller/show.js

```diff
@@ -0,0 +1,34 @@
+const showInput = require('../cli/input/show');
+const showOutput = require('../cli/output/show');
+const showModel = require('../model/show');
+
+const showValidate = require('./validate/show');
+
+const onShowResults = (showResult) => {
+  showOutput.sendOutput(showResult);
+};
+
+const onShow = async (args, options, logger) => {
+  logger.moduleLog('info', 'Exit', 'Show command invoked.');
```

Comment:
This may be `logger.moduleLog('info', 'Show', 'Show command invoked.');`

Reply:
Fixing it.